src/library.coffee

```diff
@@ -104,19 +104,28 @@ class Library extends EventEmitter
 
     @components = {} # "name" -> { command: "", language: ''|null }.  lazy-loaded using load()
 
+  getComponent: (name) ->
+    # Direct match
+    return @components[name] if @components[name]
+    withoutNamespace = path.basename name
```

Comment:
Maybe we cant, since component name is inferred without prefix in library. I guess this will do for now..